PyRoute/TradeCalculation.py

```diff
@@ -608,9 +608,14 @@ def get_trade_between(self, star, target):
         except nx.NetworkXNoPath:
             return
 
+        fwd_weight = self.route_cost(route)
+        route.reverse()
+        rev_weight = self.route_cost(route)
+        route.reverse()
```

Comment:
I think you misunderstood the comment. The idea was to run the 
`route = nx.astar_path(self.galaxy.stars, star, target, Star.heuristicDistance)`  command twice. Once for `star, target` and once for `target, star` to calculate the differences of finding the routes. If you use Traveller Map to do route finding (e.g. Regina -> Vland) and reverse the path you will hit different worlds each way. I don't want to do that unless I can further optimize `astar_path()`. I really would not expect  the route cost to be different given the same path. 

Can we put this behind a feature flag (e.g. a new `--debug` flag) to enable extra checking for when fixing / testing / debugging and not when trying to run to get output?

Reply:
A debug flag is simple enough to implement - not seeing any real need to get really really fancy

Reply:
@tjoneslo , should be fixed in https://github.com/makhidkarun/traveller_pyroute/pull/34/commits/675e54faa68f0fd449e53143f505540090403eab and https://github.com/makhidkarun/traveller_pyroute/pull/34/commits/9a5e0ada6015425febbd2549ae1ff96a37c90cd4  (broke them up to make it easier to keep track of)